src/privatesend.cpp

```diff
@@ -514,14 +515,14 @@ void CPrivateSend::SyncTransaction(const CTransaction& tx, const CBlockIndex *pi
 //TODO: Rename/move to core
 void ThreadCheckPrivateSend(CConnman& connman)
 {
-    if(fLiteMode) return; // disable all Dash specific functionality
+    if(fLiteMode) return; // disable all ZOC specific functionality
 
     static bool fOneThread;
     if(fOneThread) return;
     fOneThread = true;
 
     // Make this thread recognisable as the PrivateSend thread
-    RenameThread("dash-ps");
+    RenameThread("zeroone-ps");
```

Comment:
be carefull changing thread names, did you check if this threadname is referenced anywhere else?

Reply:
yes, I did a "Find in files" with Notepad++ :
// string match at init.cpp#L1942
    // ********************************************************* Step 11d: start zeroone-ps-<smth> threads
// string match at privatesend-client.cpp#L1463
    RenameThread("zeroone-ps-client");
// string match at privatesend-server.cpp#L904
    RenameThread("zeroone-ps-server");
// string match at privatesend.cpp#L525
    RenameThread("zeroone-ps");